opal-setup/values.yaml

```diff
@@ -83,3 +86,16 @@ appValues:
         image:
           repository: registry.k8s.io/git-sync/git-sync
           tag: v4.0.0
+      secrets:
+        nameOverride: *tokenEnv
+  daskGateway:
+    enabled: true
+    appValues:
+      namespace: *opalNS
+      gateway:
+        auth:
+          jupyterhub:
+            apiTokenFromSecretName: *tokenEnv
+            apiTokenFromSecretKey: "JUPYTERHUB_API_TOKEN"
```

Comment:
do we expect this to change very often? we might be able to leave this out of the values